src/helm/benchmark/presentation/run_specs_dec2023.conf

```diff
@@ -0,0 +1,169 @@
+  {description: "math:model=text_code,subject=intermediate_algebra,level=2,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=algebra,level=2,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=prealgebra,level=2,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=geometry,level=2,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=counting_and_probability,level=2,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=precalculus,level=2,use_chain_of_thought=True", priority: 4}
+
+  {description: "math:model=text_code,subject=number_theory,level=3,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=intermediate_algebra,level=3,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=algebra,level=3,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=prealgebra,level=3,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=geometry,level=3,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=counting_and_probability,level=3,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=precalculus,level=3,use_chain_of_thought=True", priority: 3}
+
+  {description: "math:model=text_code,subject=number_theory,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=intermediate_algebra,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=algebra,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=prealgebra,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=geometry,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=counting_and_probability,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=precalculus,level=4,use_chain_of_thought=True", priority: 4}
+
+  {description: "math:model=text_code,subject=number_theory,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=intermediate_algebra,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=algebra,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=prealgebra,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=geometry,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=counting_and_probability,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=precalculus,level=5,use_chain_of_thought=True", priority: 3}
+
+  # GSM
```

Comment:
Note: Interesting to see results for GSM vs MATH. E.g. Palymra beats all oepnai models to take 1st for gsm but is the best non-openai for math (incl CoT) to get 6th
https://crfm.stanford.edu/helm/latest/?group=reasoning